src/Components/ComponentUT.h

```diff
@@ -0,0 +1,56 @@
+// MIT License
```

Comment:
A header file is not needed for test usually, because we don't share symbols with other .cc files.